config/find-eu-exit-guidance-business.yml

```diff
@@ -0,0 +1,89 @@
+---
+base_path: "/find-eu-exit-guidance-business"
+content_id: 42ce66de-04f3-4192-bf31-8394538e0734
+signup_content_id: 2818d67a-029a-4899-a438-a543d5c6a20d
+title: Find EU Exit guidance for your business
+description: ''
+name: 'Find EU Exit guidance for your business'
+locale: en
+public_updated_at: '2018-11-23T09:00:00.000+00:00'
+publishing_app: specialist-publisher
+rendering_app: finder-frontend
+details:
+  beta: false
+  document_noun: publication
+  email_filter_facets:
+  - facet_id: sector
+    facet_name: Sector
+    facet_choices:
+    - key: aerospace
+      radio_button_name: Aerospace
+      topic_name: aerospace
+      prechecked: false
+    - key: agriculture-environment-and-natural-resources
+      radio_button_name: Agriculture, environment and natural resources
+      topic_name: agriculture-environment-and-natural-resources
```

Comment:
Topic name should be human readable. It's currently used to generate the title for the email subscription.